spec/models/user_spec.rb

```diff
@@ -8,5 +8,6 @@
     it { is_expected.to validate_presence_of(:email) }
     it { is_expected.to validate_uniqueness_of(:name) }
     it { is_expected.to validate_uniqueness_of(:email) }
+    it { is_expected.to have_secure_password }
   end
```

Comment:
I would add a spec to check that by default, a user is not admin. For example:
```
it "defaults to not be an admin" do
  expect(User.new).not_to be_admin
end
```

Reply:
Aah, of course I will add the spec, thanks Elle!